api/v1alpha1/annotations.go

```diff
@@ -12,12 +12,19 @@ const (
 	AnnotationKeyOIDCGroups   = "rbac.kargo.akuity.io/groups"
 	AnnotationKeyOIDCSubjects = "rbac.kargo.akuity.io/sub"
 
-	AnnotationKeyEventActor               = "event.kargo.akuity.io/actor"
-	AnnotationKeyEventProject             = "event.kargo.akuity.io/project"
-	AnnotationKeyEventPromotionName       = "event.kargo.akuity.io/promotion-name"
-	AnnotationKeyEventPromotionCreateTime = "event.kargo.akuity.io/promotion-create-time"
-	AnnotationKeyEventFreightAlias        = "event.kargo.akuity.io/freight-alias"
-	AnnotationKeyEventFreightName         = "event.kargo.akuity.io/freight-name"
-	AnnotationKeyEventStageName           = "event.kargo.akuity.io/stage-name"
-	AnnotationKeyEventAnalysisRunName     = "event.kargo.akuity.io/analysis-run-name"
+	AnnotationKeyEventActor                    = "event.kargo.akuity.io/actor"
+	AnnotationKeyEventProject                  = "event.kargo.akuity.io/project"
+	AnnotationKeyEventPromotionName            = "event.kargo.akuity.io/promotion-name"
+	AnnotationKeyEventPromotionCreateTime      = "event.kargo.akuity.io/promotion-create-time"
```

Comment:
```suggestion
	AnnotationKeyEventPromotionCreatedTime      = "event.kargo.akuity.io/promotion-created-time"
```

feels more natural.

```suggestion
	AnnotationKeyEventPromotionCreatedAt      = "event.kargo.akuity.io/promotion-created-at"
```

feels more specific.

Same for the `freight-create-time`, `verification-start-time`, etc. bellow.